### PR TITLE
[15.0][MIG] account_maturity_date_default: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT36425

### CODE REVIEW

File: account_maturity_date_default/tests/test_account_move.py [hunk 1]
```diff
@@ -86,26 +86,32 @@ def test_invoice_reconciliation(self):
...
-                    "name": invoice.name,
-                }
-            ]
+        statement.button_post()
+        statement_line = statement.line_ids.filtered(
+            lambda x: x.payment_ref == invoice.name
         )
-        self.assertEqual(invoice.invoice_payment_state, "paid")
-        line_receivable = statement_line.journal_entry_ids.filtered(
+        # process_reconciliation() has been moved to account_reconciliation_widget OCA.
```
Comment (pedrobaeza) [inline account_maturity_date_default/tests/test_account_move.py]: Remove these commented lines

**Review by pedrobaeza — APPROVED** [review]

**Comment by victoralmau** [comment]
Ping @chienandalu

**Review by chienandalu — APPROVED** [review]

**Pull request merged**